pkg/executor/partition_table_test.go

```diff
@@ -2407,3 +2407,26 @@ func TestIssue31024(t *testing.T) {
 
 	tk2.MustExec("rollback")
 }
+
+func TestIssue52198(t *testing.T) {
```

Comment:
Maybe we can put this kind of test case into integration test?  @Defined2014 paid a lot effect to migrate the old test cases.

See file tests/integrationtest/t/executor/partition/issues.test